### PR TITLE
Add contributors section

### PR DESCRIPTION
**Pull request by AitoApps**

### CODE REVIEW

**Comment by geekloper** [comment]
Thaanks (y) ! T9der tgad css dial contributors section, bach maykounouch lass9in ??  Bach ay wahed ykoun ma3lih ghi yzid smitou ou link dial l avatar !

image

**Comment by AitoApps** [comment]
chokran 3la Lwakte li ka Ta3tiwna wach khassni ndire pull request jdide

**Comment by AitoApps** [comment]
bdalt f l code  ou f git dart hade les cmdt:
git status
git add .
git commit -m "zat masafa bin contributors"
git push origin contributors-section

File: index.html [hunk 1]
```diff
@@ -70,21 +76,90 @@ <h4 class="subtitle is-5">Get in touch</h4>
...
+        <ul class="contributor">
+          <li>
+            <a href="https://github.com/tixboubou"><img src="https://avatars0.githubusercontent.com/u/6819197"
+                alt="Avatar" class="avatar"></a>
+          </li>
+          <Li>
+            <a href="https://github.com/tixboubou">tixboubou</a>
+          </Li>
+        </ul>
+      </div>
```
Comment (madrane9) [inline index.html:163]: should we use the same code, and put our name instead of yours?... I used the indext.hmlt but it didn't work for me?

**Comment by geekloper** [comment]
> 
> 
> bdalt f l code ou f git dart hade les cmdt:
> git status
> git add .
> git commit -m "zat masafa bin contributors"
> git push origin contributors-section

@tixboubou  Right  hadchi li kan khassek dir.

ida bghit t9der tste3mel nefss l modules li kaynin f had l CSS library li kanst3moulaha ou 7awel ma2amkan tste3mel ghi CSS dialha.. 


t9der tste3mel had l components mataln :  [Cards](https://bulma.io/documentation/components/card/)

Khoud ghi had la partie : 

image

Ila bghit tkhdem 3la hadi 7ell issue jdid ou dir fih chnu baghi tzid ou mouraha dir pull request jdid bach dak chi yji nadi 💯 

Ida jatek s3iba ana n9der nkhdem 3liha ou nta zid name nta3ek ok ? 

@tixboubou  Ok ?

File: index.html (hunk 1 above)
Comment (geekloper) [reply index.html:163]: Oui hit 3la makanden rah dar `Li` b majuscule 

@tixboubou t9der tsle7ha ? Thanks ;)

**Comment by AitoApps** [comment]
ok nhawel ndareb m3aha

**Comment by geekloper** [comment]
@tixboubou  hahia wahed l fikra, ida 3jbatek jerbha 


```HTML
  <!-- contributors -->
  <section class="section" id="contributors">
    <div class="container">
      <div class="section-contact-heading ">
        <h3 class="title is-2">Contributors</h3>
      </div>
      <div class="columns" id="contributors_columns">

      </div>
        <div class="card-content">
          <div class="media">
          <div class="media-left">
              <figure class="image is-48x48">
              <a href="https://github.com/tixboubou"  target="_blank">
                  <img src="https://avatars0.githubusercontent.com/u/6819197" alt="tixboubou">
              </a>
              </figure>
          </div>
          <div class="media-content">
              <p class="title is-4">redoine</p>
              <p class="subtitle is-6">@tixboubou</p>
          </div>
          </div>
      </div>
    </div>
  </section>
```

**Pull request closed without merging**